ic10emu/src/grammar.rs

```diff
@@ -1090,21 +668,35 @@ impl Number {
 
 #[cfg(test)]
 mod tests {
+    use color_eyre::eyre::Ok;
+    use strum::EnumProperty;
+
     use super::*;
 
+    static INIT: std::sync::Once = std::sync::Once::new();
```

Comment:
shouldn't this be a thread local?

Reply:
can tests get multi-threaded? and would that matter? I'm using it to set color_eyre's panic hook.

Reply:
I was thinking about tracing, never mind!